R/build-articles.R

```diff
@@ -395,9 +395,9 @@ data_articles_index <- function(pkg = ".") {
   missing <- missing[!article_is_intro(missing, package = pkg$package)]
 
   if (length(missing) > 0) {
-    cli::cli_abort(
-      "{length(missing)} vignette{?s} missing from index in \\
-      {pkgdown_config_href({pkg$src_path})}: {.val {missing}}.",
+    config_abort(
+      pkg,
+      "{length(missing)} vignette{?s} missing from index: {.val {missing}}",
```

Comment:
```suggestion
      "{length(missing)} vignette{?s} missing from index: {.val {missing}}.",
```

Noticed this via the snapshot :nail_care: